app/jobs/recording_ready_notifier_job.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'jwt'
+require 'net/http'
+
+class RecordingReadyNotifierJob < ApplicationJob
+  queue_as :call_back
+
+  LOGGER = Delayed::Worker.logger
```

Comment:
All capitals?

Reply:
Inside a Rails ApplicationJob, you should be able to use the normal rails logger? Shouldn't be any need for this, I think…

(If there is a need to set up the logger specially for delayed job, it should be done in an initializer or in the delayed job startup script, not in the job itself)